e2e/tests/node_reboot.robot

```diff
@@ -0,0 +1,31 @@
+*** Settings ***
+Documentation    Negative Test Cases
+Resource    ../keywords/volume.resource
+Resource    ../keywords/node.resource
+
+Test setup    set_test_environment
+Test Teardown    cleanup_resources
+
+*** Variables ***
+${LOOP_COUNT}    1
+
+*** Test Cases ***
+Replica Rebuilding While Volume Attached Node Reboot
```

Comment:
Let's make the test name concise.

`Reboot the volume node while the replica is being rebuilt`

Reply:
Modified to `Reboot Volume Node While Replica Rebuilding`.